src/main/java/Duke.java

```diff
@@ -1,10 +1,143 @@
+import java.util.*;
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
+    private Task task;
+
+    public static void main(String[] args) throws DukeException {
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?");
+        Duke newDuke = new Duke();
+        newDuke.chatDuke();
+
+
+        /*String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello from\n" + logo);*/
     }
+
+    public void chatDuke() throws DukeException {
+        Scanner sc = new Scanner(System.in);
+        ArrayList<Task> taskArray = new ArrayList<>();
+        String str = sc.nextLine();
+        int counter = 0;
```

Comment:
Consider using ArrayList.size() to replace an explicit counter variable to get the last task in the list.